target/linux/ath79/dts/qca9531_mikrotik_routerboard-wapr-2nd.dts

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca9533_mikrotik_routerboard-16m.dtsi"
```

Comment:
Since this device is qca9531, one should rename the dtsi to "qca953x_mikrotik_routerboard-16m.dtsi" and update all references in a separate commit before adding the wAP R. Just rebase and push it here ...

Reply:
Hmmm... despite what [MikroTik's product page](https://mikrotik.com/product/RBwAPR-2nD) says, it seems it's a QCA9533:
```
root@OpenWrt:~# dmesg | grep 953
[    0.000000] Linux version 5.4.75 (compiler@server) (gcc version 8.4.0 (OpenWrt GCC 8.4.0 r14953-4d36569b9c)) #0 Mon Nov 16 16:20:40 2020
[    0.000000] SoC: Qualcomm Atheros QCA9533 ver 2 rev 0
[   14.111102] ieee80211 phy0: Atheros AR9531 Rev:2 mem=0xb8100000, irq=13
```
The SoC has a heat sink, so I can't check the actual model. I'll change the file name and the SoC in the image Makefile to qca9533. Still, should I rename the .dtsi?

Reply:
> The SoC has a heat sink, so I can't check the actual model. I'll change the file name and the SoC in the image Makefile to qca9533. Still, should I rename the .dtsi?

If the device is actually qca9533, I wouldn't touch the DTSI name until we need to for a later device.